Sources/TuistKit/Generator/ConfigGenerator.swift

```diff
@@ -165,6 +170,21 @@ final class ConfigGenerator: ConfigGenerating {
             settings["MACH_O_TYPE"] = "staticlib"
         }
 
+        if target.product.testsBundle {
+            let appDependencies = graph.targetDependencies(path: sourceRootPath, name: target.name).filter { targetNode in
+                targetNode.target.product == .app
+            }
+
+            if let app = appDependencies.first {
+                settings["TEST_TARGET_NAME"] = "\(app.target.name)"
```

Comment:
looks like Xcode only populates `TEST_TARGET_NAME` for ui tests only - it doesn't seem to mind it being there, guess this helps simplify `generateTestTargetIdentity`

Reply:
I will leave it, keeps it simple.